desktop-exporter/app/components/detail-view/fields-pannel.tsx

```diff
@@ -0,0 +1,200 @@
+import React from "react";
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  TabPanel,
+  Box,
+  AccordionIcon,
+  Divider,
+  List,
+  Heading,
+} from "@chakra-ui/react";
+
+import { SpanData } from "../../types/api-types";
+import { SpanField } from "./span-field";
+import { getDurationString } from "../../utils/duration";
+
+type FieldsPannelProps = {
+  span: SpanData | undefined;
+};
+
+export function FieldsPannel(props: FieldsPannelProps) {
+  let { span } = props;
+  if (!span) {
+    return (
+      <TabPanel>
+        <p>Nothing here yet.</p>
+      </TabPanel>
+    );
+  }
+  let durationString = getDurationString(span.startTime, span.endTime);
+
+  let spanAttributes = Object.entries(span.attributes).map(([key, value]) => (
```

Comment:
This is going to iterate over them in a random order. We should explore sorting them alphabetically before rendering in a follow-on PR.